### PR TITLE
Fetch the countries and jurisdiction for front end

### PR DESCRIPTION
**Pull request by mariamera**

### CODE REVIEW

File: paradise_papers_search/static/js/search.js [hunk 1]
```diff
@@ -184,14 +184,15 @@
    */
   class SearchApp {
     constructor() {
+
```
Comment (raulgomezc) [inline paradise_papers_search/static/js/search.js]: Surplus line here

File: paradise_papers_search/static/js/search.js [hunk 2]
```diff
@@ -237,8 +240,39 @@
...
+        });
+      })
+      .fail(() => {
+        /*TODO Handle errors */
+        console.log("Fetch error");
+      })
+    }
   }
 
+
```
Comment (raulgomezc) [inline paradise_papers_search/static/js/search.js]: Surplus line here

File: paradise_papers_search/static/js/search.js [hunk 3]
```diff
@@ -237,8 +240,39 @@
         this._currentNodeSearch.fetch();
       }
     }
+
+    fetchCountry() {
```
Comment (raulgomezc) [inline paradise_papers_search/static/js/search.js]: Change the name of this function to fetchCountries to match the one below

**Review by raulgomezc — CHANGES_REQUESTED** [review]

File: paradise_papers_search/static/js/search.js [hunk 4]
```diff
@@ -202,6 +203,8 @@
       }
 
       this._currentNodeSearch = this._nodeSearchList()[0];
+      this.fetchCountry()
```
Comment (raulgomezc) [inline paradise_papers_search/static/js/search.js]: Missing semicolon here

File: paradise_papers_search/static/js/search.js [hunk 5]
```diff
@@ -202,6 +203,8 @@
...
+      this.fetchJurisdictions()
```
Comment (raulgomezc) [inline paradise_papers_search/static/js/search.js]: Same here

**Review by raulgomezc — CHANGES_REQUESTED** [review]

File: paradise_papers_search/static/js/search.js [hunk 6]
```diff
@@ -183,15 +183,16 @@
...
   class SearchApp {
+
     constructor() {
       this._searchText = ko.observable('');
       this._initialSearchDone = ko.observable(false);
-
-      // TODO
-      this._countryList = ko.observableArray(mockup_data.countries);
-      this._jurisdictionList = ko.observableArray(mockup_data.jurisdictions);
+      this._searchText = ko.observable('');
```
Comment (junior1q94) [inline paradise_papers_search/static/js/search.js:189]: Duplicated _searchText observable definition.

**Review by junior1q94 — CHANGES_REQUESTED** [review]

**Review by junior1q94 — APPROVED** [review]

**Review by oscarpolanco — APPROVED** [review]
LGTM

**Pull request merged**